modules/programs/ssh.nix

```diff
@@ -228,6 +228,14 @@ in
         the known hosts file.
       '';
     };
+    
+    host = mkOption {
```

Comment:
I like the idea but the name `host` is a bit vague I think. Maybe `fallbackHost`, `finalHost`, or something?